src/main/resources/application.yml

```diff
@@ -8,14 +14,8 @@ spring:
     stream:
       bindings:
         run-task:
-          binder: rabbit
-          destination: core-valid-run-task
-          group: job-launcher
-      rabbit:
-        default:
-          consumer:
-            auto-bind-dlq: true
-            max-attempts: 1
+          destination: cse-d2cc-run-task
+          group: cse-d2cc-job-launcher
```

Comment:
Why did you change that ?

Reply:
with comparing to the deployment config i assumed that it is not useful .. i guess forgotten after changing function by consumer?
For the destination and group i didn't gave it too much interest as it have to be reconfigured any way in dev mode. Also overridden in prod mode..

Reply:
Yes it is not overriden in the prod files but it is required in the build of the application otherwise it wouldn't be taken into account. We don't have to redefine these parameters everytime in the prod deployments because there are defined here. So there are required. Maybe not for the "binder: rabbit" we can test but the other ones are not the default values for rabbit binder so we need them.